docs/DeveloperGuide.md

```diff
@@ -238,62 +238,64 @@ _{more aspects and alternatives to be added}_
 
 _{Explain here how the data archiving feature will be implemented}_
 
-### \[Proposed\] Branch feature
+### Check-out feature
```

Comment:
Since the command name is checkout, is it better to remove the hyphen here?

Reply:
Changed spelling of Check-out to checkout